### ISSUE TITLE
Not Working With WordPress 5.5

### ISSUE THREAD

**Issue by tosunkaya** [CMB2/CMB2#1367]
CMB2 isn't working with WordPress 5.5 Please update.

They updated fields on the new version.

**Comment by tw2113** [comment]
In what way is it not working for you? Not saving some things? errors being thrown just by loading a page with some CMB2 build fields in place? Something else?

I checked one of my own personal sites that makes heavy use of CMB2 latest, and is on WP 5.5 and wasn't seeing anything the matter. So any information you could provide for what you're seeing would be a big help.

**Comment by tosunkaya** [comment]
I can't click any button like change public date, publicity, some JS problem. Even couldn't disable CMB2 from plugins page. WordPress 5.5 is too problematic. Downgraded to 5.4.2 fully works.

**Comment by jtsternberg** [comment]
Not seeing any issues. It's likely a conflict w/ the theme or other plugins, or custom code. You'll have to look in your debug/error logs to see what's happening.

**Comment by huxh123123** [comment]
![Screenshot (110)](https://user-images.githubusercontent.com/60230513/90141881-5cc41f80-dd84-11ea-8fe9-8bd5b1f7eacf.png)

**Comment by huxh123123** [comment]
![Screenshot (111)](https://user-images.githubusercontent.com/60230513/90141921-69487800-dd84-11ea-934c-440dc491822a.png)

**Comment by jtsternberg** [comment]
`CMB2_ICON` is not a thing in CMB2 at all. Again, looks like another plugin, the theme, or custom code is the culprit.
https://github.com/CMB2/CMB2/search?q=CMB2_ICON&unscoped_q=CMB2_ICON

**Comment by moisb** [comment]
I have the same problem. Everything was working perfectly until today I upgraded to Wordpress 5.5, after that (I did not update any other plugins) the CMB2 fields stopped working.

Until now i realized that the fields stopped working:
multiselect option
text_date (date picker)

On console I see an error related to the icon.js file:
wp-coupon/inc/metabox-addons/icon/icon.js?ver=true

Uncaught ReferenceError: CMB2_ICON is not defined
    at HTMLDocument.<anonymous> (icon.js?ver=true:6)
    at i (load-scripts.php?c=0&load[chunk_0]=jquery,utils&ver=5.5:2)

I saw that this error also conflicts (break) fields created by Advanced Custom Field and the Metabox of Yoast SEO: https://pasteboard.co/JmtZY0a.jpg

My theme is WP Coupon from Famethemes.
